operator/src/main/java/oracle/kubernetes/operator/work/Fiber.java

```diff
@@ -318,6 +319,22 @@ public boolean isDone() {
     return status.get() == DONE;
   }
 
+  /**
+   * The most recently invoked step if the fiber is currently suspended.
+   * @return Last invoked step for suspended fiber.
+   */
+  public Step getLastIfSuspended() {
```

Comment:
The name is a bit unclear. Maybe `getSuspendedStep` would be clearer. Its semantics would be the same, as a fiber can have at most one suspended step.